src/physics/base/Process.hh

```diff
@@ -19,6 +19,17 @@ namespace celeritas
 {
 class Model;
 
+//---------------------------------------------------------------------------//
+/*!
+ * Type of physics process.
+ */
+enum class ProcessType
+{
+    electromagnetic_discrete, //!< Discrete EM process
+    electromagnetic_dedx,     //!< Continuous-discrete EM process
+    electromagnetic_msc       //!> Multiple scattering
```

Comment:
Looks like there was typo here. `!>` instead of `!<`.

Reply:
haha oops, good catch!